principia_make.sh

```diff
@@ -1,9 +1,12 @@
-./autogen.sh
-./configure \
-    CC=clang \
-    CXX=clang++ \
-    CFLAGS="${C_FLAGS?}" \
-    CXXFLAGS="${CXX_FLAGS?}" \
-    LDFLAGS="${LD_FLAGS?}" \
-    LIBS="-lc++ -lc++abi"
+mkdir -p build
+pushd build
+cmake \
+    -DCMAKE_C_COMPILER:FILEPATH="$(which clang)" \
+    -DCMAKE_CXX_COMPILER:FILEPATH="$(which clang++)" \
+    -DCMAKE_C_FLAGS="${C_FLAGS}" \
+    -DCMAKE_CXX_FLAGS="${CXX_FLAGS}" \
```

Comment:
`?` before the `}` as was done in the original.  Same at the next line.

Reply:
Fixed.